modules/pel/peltool/peltool.py

```diff
@@ -504,11 +515,34 @@ def extractAllPELsData(path: str, config: Config, extension: str, rev: bool = Fa
                 stream = DataStream(data, byte_order='big', is_signed=False)
                 _, json_string = parsePEL(stream, config, False)
                 if json_string:
-                    allPELsData = allPELsData + json_string + ",\n"
+                    if not print_hex:
+                        allPELsData = allPELsData + json_string + ",\n"
+                    else:
+                        printPELInHexFormat(data)
             except Exception as e:
                 print(f"Exception: No PEL parsed for {file}: {e}")
-    allPELsData = allPELsData[:-2] + "\n]"
-    print(allPELsData)
+    if not print_hex:
+        allPELsData = allPELsData[:-2] + "\n]"
+        print(allPELsData)
+
+
+def printPELInHexFormat(data: memoryview) -> None:
+    """
+    Read single PEL file data and display in hexdecimal format.
+    Returns: None
+    Prints PEL data in the hexdecimal format.
+    """
+    try:
+        mv = memoryview(data)
+        hexdata = hexdump(mv)
+        # Add line seperator for PELs in hexadecimal format.
+        print("-------------- PEL Begin  ----------------")
```

Comment:
I don't think we'd need the Begin/End statements?  If a user wanted that, they could just add that themselves.

Reply:
I believe it would be helpful to provide some indication between PELs when the user uses the -l/-a options. Otherwise, it will appear as continuous byte values, making it difficult to understand or identify where a PEL begins and ends.

Reply:
Yes, especially where there are more than 10 pels. It is getting tough to identify the begin n end of the pels in hexa format. Line separator is good to distinguish.

Reply:
Yea, fair enough I guess.